tools/cgpatch/README.md

```diff
@@ -0,0 +1,23 @@
+# CGPatch
```

Comment:
If this has a paper somewhere, I think this README would be a good place to add a reference to that paper.